.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "singleQuote": true,
   "trailingComma": "none",
+  "tabWidth": 2,
```

Comment:
Why did you add it back?

Reply:
The default setting is 2, but without adding it, it won't automatically apply the style.

Reply:
Ah, nice catch!